static/src/javascripts/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -13,8 +13,13 @@ define([
             variants: ['control', 'member-contribute']
         };
 
+        var contributionsMembershipEpicSideBySide = {
+            name: 'ContributionsMembershipEpicSideBySide',
+            variants: ['control', 'member-contribute']
```

Comment:
does this variant name make sense for this test? (just that it's the same as the other test)

you will know better than I do...